implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -771,7 +771,9 @@ impl NodeManagerWorker {
             (Delete, ["v0", "spaces", id]) => self.delete_space(ctx, dec, id).await?,
 
             // ==*== Projects ==*==
-            (Post, ["v0", "projects", space_id]) => self.create_project(ctx, dec, space_id).await?,
+            (Post, ["v1", "spaces", space_id, "projects"]) => {
+                self.create_project(ctx, dec, space_id).await?
+            }
```

Comment:
Just curious on the version bumping here and the rest below.

What is the reasoning behind updating these to v1?

Reply:
My thoughts regarding this are - would still leave this at v0 since we're removing the existing v0 endpoint. 

There is no version managing happening, and all internal callers are updated.

Then we can bump everything to v1 when we're establishing to be ready.

Reply:
(Also helps remove any confusion as to why some are v1 and some are v0)